tests/lib/rules/no-extra-parens.js

```diff
@@ -291,7 +314,61 @@ ruleTester.run("no-extra-parens", rule, {
         { code: "foo instanceof (bar instanceof baz)", options: ["all", { nestedBinaryExpressions: false }] },
         { code: "foo in (bar in baz)", options: ["all", { nestedBinaryExpressions: false }] },
         { code: "foo + (bar + baz)", options: ["all", { nestedBinaryExpressions: false }] },
-        { code: "foo && (bar && baz)", options: ["all", { nestedBinaryExpressions: false }] }
+        { code: "foo && (bar && baz)", options: ["all", { nestedBinaryExpressions: false }] },
+
+        // ignoreJSX === "all"
+        jsx("const Component = (<div />)", ["all", { ignoreJSX: "all" }]),
```

Comment:
Thanks for the suggestion. Using the defaults for `RuleTester` sound like the best way to do this. Did not know this is possible. Updated.